Modules/CaPTkSurvival/include/CaPTkClassifierUtils.h

```diff
@@ -0,0 +1,296 @@
+#pragma once
+
+#include "CaPTkDefines.h"
+#include "CaPTkEnums.h"
+#include "opencv2/core/core.hpp"
+#include "opencv2/ml.hpp"
+/**
+\brief Train and save the SVM classifier
+
+\param trainingDataAndLabels Input training data with last column as training labels
+\param outputModelName File to save the model
+*/
+inline VectorDouble trainOpenCVSVM(const VariableSizeMatrixType &trainingDataAndLabels, const std::string &outputModelName, bool considerWeights, int ApplicationCallingSVM)
+{
+  cv::Mat trainingData = cv::Mat::zeros(trainingDataAndLabels.Rows(), trainingDataAndLabels.Cols() - 1, CV_32FC1),
+    trainingLabels = cv::Mat::zeros(trainingDataAndLabels.Rows(), 1, CV_32FC1);
+
+  //// debugging purposes
```

Comment:
commented code

Reply:
The commented-out code has been removed.